src/main/java/com/airlines/dao/CrewMemberDaoImpl.java

```diff
@@ -0,0 +1,106 @@
+package com.airlines.dao;
+
+import com.airlines.exception.DaoOperationException;
+import com.airlines.model.Citizenship;
+import com.airlines.model.CrewMember;
+import com.airlines.model.Position;
+
+import java.sql.Connection;
+import java.sql.Date;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.Objects;
+
+public class CrewMemberDaoImpl implements CrewMemberDao {
+
+    private final static String INSERT_CREW_MEMBER_SQL =
+            "INSERT INTO crew_member(first_name, last_name, position, birthday, citizenship, crew_id, id) " +
```

Comment:
a little weird for me placement for main id:
try to follow database structure so not confuse and have code reading more simple:
INSERT INTO crew_member(id, first_name....etc

Reply:
done in https://github.com/pielena/airlines-jdbc-app/commit/8f61d1a09ce2d8aa9876fbe7bcce6e05b7a32a08